custom_components/ha-glocaltokens/api.py

```diff
@@ -46,3 +57,69 @@ def get_google_devices_json(self):
     def get_android_id(self):
         """Generate random android_id"""
         return self._client._get_android_id()
+
+    def format_timer_information(timer):
+        timestamp_ms = timer[FIRE_TIME]
+        timestamp = timestamp_ms / 1000
+        humantime = datetime.fromtimestamp(timestamp).strftime(SHOW_TIME_ONLY)
+        timer[DATE_TIME] = humantime
+
+        duration_ms = timer[ORIGINAL_DURATION]
+        duration = duration_ms / 1000
+        humanduration = datetime.utcfromtimestamp(duration).strftime(SHOW_TIME_ONLY)
+        timer[DURATION] = humanduration
+        return timer
+
+    def format_alarm_information(alarm):
+        timestamp_ms = alarm[FIRE_TIME]
+        timestamp = timestamp_ms / 1000
+        humantime = datetime.utcfromtimestamp(timestamp).strftime(SHOW_DATE_TIMEZONE)
+        localtime = datetime.fromtimestamp(timestamp).strftime(SHOW_DATE_AND_TIME)
+        alarm[DATE_TIME] = humantime
+        alarm[LOCAL_TIME] = localtime
+        return alarm
+
+    def create_url(ip):
+        url = 'https://'+ ip +':'+ str(PORT) + ENDPOINT
```

Comment:
Avoid using string casting like this. A more pythonic way would be like this:
```python
url = 'https://{ip}:{port}/{endpoint}'.format(
         ip=ip, 
         port=PORT,
         endpoint=ENDPOINT
)
```
You could also use f-string, but those aren't backwards compatible with python2, so `format` is a safe way to go :)

Reply:
I will do that